### PR TITLE
Joystick tests

### PR DESCRIPTION
**Pull request by kacperzolkiewski**

### CODE REVIEW

**Review by adam-kostuch — COMMENTED** [review]
It would be great if You could create `__tests__` folder in `src/joystick` directory, just to keep all of the tests in there

**Review by kgutka — CHANGES_REQUESTED** [review]
Add `__tests__` directory and put `joystick.spec.ts` inside it
- I didn't see the previous comment about the same issue. Please do it, pull recent changes from master and then we will merge

**Review by adam-kostuch — APPROVED** [review]

**Review by adam-kostuch — APPROVED** [review]

File: src/boot/boot.ts [hunk 1]
```diff
@@ -31,6 +31,17 @@ export class Boot extends Entity {
 
       // validate if player can move there (walls) - TODO
```
Comment (adam-kostuch) [inline src/boot/boot.ts:32]: We have to delete all of the remaining comments.

**Review by adam-kostuch — CHANGES_REQUESTED** [review]

File: src/boot/boot.ts (hunk 1 above)
Comment (kacperzolkiewski) [reply src/boot/boot.ts]: ok

**Review by kgutka — APPROVED** [review]

**Review by adam-kostuch — APPROVED** [review]

**Review by adam-kostuch — CHANGES_REQUESTED** [review]
Wait with merging for `update-code` to be merged first. Then rebase changes, because i moved `src/node/__tests__/node.mock.ts` and `src/node/__tests__/index.ts` into new `./mocks` directory. 😄

**Comment by kgutka** [comment]
Closes #55

File: src/Player/components/draw/draw.ts [hunk 2]
```diff
@@ -25,13 +25,15 @@ export class PlayerDrawComponent implements IComponent {
...
+      this.Draw()
+    }
   }
 
   private Draw(): void {
     CanvasLayer.Foreground.DrawPlayer(this.Position, this.Entity.Team)
-    CanvasLayer.Foreground2.FillBlock()
-    CanvasLayer.Foreground2.FillGround()
+    // CanvasLayer.Foreground2.DrawBlocks()
+    // CanvasLayer.Foreground2.DrawGrounds()
```
Comment (adam-kostuch) [inline src/Player/components/draw/draw.ts:36]: If those parameters won't be used later, we should delete them

**Review by adam-kostuch — CHANGES_REQUESTED** [review]

File: src/Player/components/draw/draw.ts (hunk 2 above)
Comment (kgutka) [reply src/Player/components/draw/draw.ts]: done

**Review by adam-kostuch — APPROVED** [review]

**Pull request merged**